support-frontend/assets/pages/[countryGroupId]/oneTimeCheckout.tsx

```diff
@@ -114,7 +142,91 @@ function OneTimeCheckoutComponent({ geoId }: Props) {
 					</div>
 				</BoxContents>
 			</Box>
-			<form></form>
+			<form
+				ref={formRef}
+				action="todo"
+				method="POST"
+				onSubmit={() => {
+					/* ToDo */
+				}}
+			>
+				<Box cssOverrides={shorterBoxMargin}>
+					<BoxContents>
+						<FormSection>
+							<Legend>1. Your details</Legend>
+							<div>
```

Comment:
What's this `div` do?

Reply:
Can't see any reason for it - copy paste from `checkout.tsx`. Let's get rid

Reply:
![image](https://github.com/user-attachments/assets/c1902caf-48c0-465f-8094-d38488cdefc7)

The div is for styling purposes - the style in `formSection` is adding top margin to the not first in type, which the `div` is. Maybe we can do this in a more obvious way but let's come back to it